GitCommands/Settings/AppSettings.cs

```diff
@@ -691,6 +691,7 @@ public enum PullAction
             Merge,
             Rebase,
             Fetch,
+            FetchPrune,
```

Comment:
❗️ This will have a profound effect on the default pull action dropdown. 
We can't do that.

![image](https://user-images.githubusercontent.com/4403806/53674952-e6457600-3ce5-11e9-849d-fd321c4b3062.png)

Reply:
I didn't find any problem (yet). What I tested so far:
1) set all available default pull actions and verify that they are saved correctly in GitExtensions.settings item `DefaultPullAction` -> ok
2) set all available default pull actions and verify that they are applied correctly after clicking `Open pull dialog ...` and the dialog is setup correctly -> ok
3) trigger all available options from the pull dialog dropdown and verify that they execute the expected git command -> ok
4) trigger all available options from the left panel and verify that they execute the expected git command -> ok
5) Extended unit tests `Should_correctly_setup_for_defined_pull_action` and `Should_use_user_DefaultPullAction_pull_action_None` wich cases for `PullAction.FetchPrune` -> ok

Do I miss something?

Would it be better to add the new "Fetch and prune" to the default pull action dropdown as well?

Reply:
These options are also used when the app is invoked via the shell extensions.
We had a number of issues associated with this area, e.g. #6144, #6194, #6200; and I wouldn't want a repeat of these...

Reply:
I force pushed changes to remove `PullAction.FetchPrune`.